server/notification-providers/techulus-push.js

```diff
@@ -10,11 +10,22 @@ class TechulusPush extends NotificationProvider {
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
         const okMsg = "Sent Successfully.";
 
+        let data = {
+            "title": notification?.pushTitle?.length ? notification.pushTitle : "Uptime-Kuma",
```

Comment:
Please add this default to the frontend as well.

Here is how this works

https://github.com/louislam/uptime-kuma/blob/e8c650797c80fe7189091afa9a81d4bad679dac1/src/components/notifications/Ntfy.vue#L66-L80

Reply:
Thanks, I've added this.